backend/capellacollab/sessions/hooks/t4c.py

```diff
@@ -81,14 +81,15 @@ def configuration_hook(
 
         for repository in t4c_repositories:
             try:
+                token_username = auth_injectables.get_username(token)
```

Comment:
I would prefer simply naming this `username` here as we do it at all other places. 
In addition, we can't simply use it like this. The `get_username` function expects a fastapi request and not the token. So I would simply inject the username instead of the token in the `request_persistent_session` route and change the configuration hook to expect the username directly and not the token.

Reply:
good point